brat/Data/Hugr.hs

```diff
@@ -468,6 +502,8 @@ partialOp parent funcSig numSupplied = CustomOp parent "Brat" "Partial" sig args
   otherInputs = drop numSupplied (input funcSig)
 
 
+-- N.B. We don't need an implementation of LoadFunction because we're not using
+-- hugr's type parameters.
```

Comment:
But you're still implementing it below?

I think it's still good to keep `LoadFunction` so we can also load a function from a `FunctionDef` instead of a Hugr constant

Reply:
Comment is out of date 😅